content/en/docs/setup/production-environment/windows/intro-windows-in-kubernetes.md

```diff
@@ -327,6 +327,12 @@ In a cluster that includes Windows nodes, you can use the following types of Ser
   * `LoadBalancer`
   * `ExternalName`
 
+{{< warning >}}
+There are known issues with `NodePort` services on Windows Server 2022 for l2bridge networking when DSR is enabled and for all overlay networking configurations.
```

Comment:
There are known issue with NodePort services on overlay networking, if the target destination node is Windows. To avoid the issue entirely, you can configure the service with externalTrafficPolicy:Local.

There are known issue with pod to pod connectivity on l2bridge network on Windows Server 2022 with KB5005619 or higher installed. To workaround the issue and restore pod-pod connectivity, you can disable the WinDSR feature in kube-proxy.

Reply:
Thanks for the clarification let me make some updates.

Reply:
Updated, thanks David!